roles/haproxy/config/tasks/main.yml

```diff
@@ -39,6 +42,7 @@
     state: restarted
   when: haproxy_syslog_conf is changed
 
```

Comment:
There are four comments starting with XXX that doesn't go along with the rest of file. I think we should remove them from this changeset.

other than that, lgtm!